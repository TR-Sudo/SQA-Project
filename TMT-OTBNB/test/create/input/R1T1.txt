Login
AdminUser
create
username11
AA
Logout
quit