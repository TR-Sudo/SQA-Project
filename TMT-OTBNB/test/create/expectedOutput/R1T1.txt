Enter transaction: Enter username: You have been logged in
Enter transaction: Enter username: Enter account type: Created user username11 with account type AA
Enter transaction: You have been logged out
Enter transaction: 